src/or/dirauth/dirvote.c

```diff
@@ -3899,3 +3899,16 @@ dirvote_parse_sr_commits(networkstatus_t *ns, smartlist_t *tokens)
   }
 }
 
+/* For the given vote, free the shared random commits if any. */
+void
+dirvote_free_commits(networkstatus_t *ns)
```

Comment:
I think this function should have a name like "clear_commits" or "remove_commits", not "free_commits", since it doesn't take the commits as an argument.

Reply:
Done in fixup: 3300369875